features/steps/notification_service_dependencies.py

```diff
@@ -26,64 +28,69 @@
 
 @when("I retrieve data from insights-content-service on {host:w}:{port:d}")
 @given("insights-content service is available on {host:w}:{port:d}")
-def check_content_service_availability(context, host, port):
+@then("I should get data from insights-content-service")
+def check_content_service_availability(context, host = None, port = None):
     """Check if insights-content-service is available at given address."""
+    assert (host is not None and port is not None) or \
+           (hasattr(context, "content_host") and hasattr(context, "content_port")), \
```

Comment:
test for None is not enough there?

Reply:
yep, I don't get this check. Do we care about `context` having these keys? we are not accessing them in the whole function

Reply:
The context here is for storing host and port if it has been used in one of the previous steps. There is annotation `@then("I should get data from insights-content-service")` (which was originally separate function, that doesn't have host and port). Either this, or I will put those arguments also to this annotation.

Reply:
@juandspy They are accessed on lines 39 and 40.

Reply:
sure, but why not just test for None?

Reply:
Yep, I see it now. Will change that.